Xamarin.PropertyEditing.Mac/Controls/Custom/PropertyTextField.cs

```diff
@@ -11,5 +11,11 @@ public PropertyTextField ()
 			Cell.LineBreakMode = NSLineBreakMode.TruncatingTail;
 			Cell.UsesSingleLineMode = true;
 		}
+
+		public override bool BecomeFirstResponder ()
+		{
+			ScrollRectToVisible (this.Bounds);
```

Comment:
We don't need `this` here and we should only actually scroll if `base.Become..` returns true.

Reply:
From testing `BecomeFirstResponder ()` is only called when the control is about to receive focus. What scenarios have you seen where `BecomeFirstResponder ()` is called and our property controls should *not* scroll into view? @alanmcgovern Any insights here? I'm happy to make the change, but want to understand exactly why the check is absolutely necessary.

Reply:
If you read the docs on `BecomeFirstResponder`, you'll see that the return value is whether or not it actually became first responder. If it hasn't actually become first responder, we don't want to scroll to it. It doesn't matter whether we've seen it happen or not, implementing it correctly is a defensive measure.

Reply:
I had already changed it, but hadn't pushed. Done.